examples/serial_relay/serial_relay.ino

```diff
@@ -1,15 +1,15 @@
 #include <A6lib.h>
 
 // Instantiate the library with TxPin, RxPin.
-A6lib A6l(D6, D5);
```

Comment:
That's really odd, I thought they defined aliases for their pins. Can we change them to `(12, 14)` so they're compatible with the ESP as well?